### PR TITLE
Fix missing message.properties file + remove .git folder if operation fails

### PR DESCRIPTION
**Pull request by blankg**

### CODE REVIEW

**Comment by skabashnyuk** [comment]
ok. @vinokurig ?

File: che-core-git-impl-jgit/src/main/java/org/eclipse/che/git/impl/jgit/JGitConnection.java [hunk 1]
```diff
@@ -583,13 +595,32 @@ public void init(InitRequest request) throws GitException {
...
+            }
+
             throw new GitException(e.getMessage(), e);
         }
     }
 
+    private void deleteRepositoryFolder() {
+        try {
+            if (repository.getDirectory().exists()) {
+                FileUtils.delete(repository.getDirectory(), FileUtils.RECURSIVE | FileUtils.IGNORE_ERRORS | FileUtils.IGNORE_ERRORS);
```
Comment (vinokurig) [inline che-core-git-impl-jgit/src/main/java/org/eclipse/che/git/impl/jgit/JGitConnection.java]: why do you use `FileUtils.IGNORE_ERRORS` twice?

**Comment by blankg** [comment]
Removed the duplicate IGNORE_ERRORS flag

**Comment by vinokurig** [comment]
ok for me

File: che-core-git-impl-jgit/src/main/resources/org.eclipse.che.git.impl.jgit/messages.properties [hunk 2]
```diff
@@ -0,0 +1,34 @@
+#
```
Comment (vinokurig) [inline che-core-git-impl-jgit/src/main/resources/org.eclipse.che.git.impl.jgit/messages.properties]: Properties file pthname is invalid, it should be: `che-core-git-impl-jgit/src/main/resources/org/eclipse/che/git/impl/jgit/messages.properties`

**Comment by blankg** [comment]
properties path should work now

**Pull request merged**